forwards_paper.tex

```diff
@@ -822,46 +822,42 @@ \subsubsection*{Computational complexity}
-% near-linear in $\rho$ (Figure \ref{fig:runtimes_selection}).  
-In this section, we explore the how required computational effort and storage space scale
+% near-linear in $\rho$ (Figure \ref{fig:runtimes_selection}).
+In this section, we explore how the required computational effort and storage space scale
 in a Wright-Fisher model with recombination.
 
 %% Memory reduction thanks to simplification
-Consider a simulation of a Wright-Fisher population of $2N$ haploid individuals
+Consider a simulation of a Wright-Fisher population of $N$ haploid individuals
 using Algorithm~\algref{W} for $T$ generations,
 with simplify interval $s = 1$ so that redundant information is removed after every generation.
 Since every chromosome inherits material from both parents,
 without simplification this would produce tables of
-$2NT$ nodes and
-$4NT$ edges.
-Suppose that $T$ is large enough that all samples coalesce within the simulation with high probability,
-($T = 20N$, say).
+$NT$ nodes and
+$2NT$ edges.
+Suppose that $T$ is large enough that all samples coalesce within the simulation with high probability
+($T = 20N$, say, giving a total space requirement of $60N^2$).
 After simplification, we are left with the tree sequence describing the history
-of only the current generation of $2N$ individuals.
-If no coalescent events involve more than two lineages, this tree sequence has $4N-2$ edges to describe the leftmost tree;
+of only the current generation of $N$ individuals.
+If no coalescent events involve more than two lineages, this tree sequence has $2N-2$ edges to describe the leftmost tree;
 and each time the marginal tree changes along the sequence,
 four edges end and four new edges begin
 (except for changes affecting the root, which require fewer; see \citet{kelleher2016efficient}).
 Coalescent theory tells us that
-the expected total length of the edges of a marginal tree is approximately $4N\log(2N)$.
+the expected total length of the edges of a marginal tree is approximately $2N\log(N)$.
 Since we have taken one crossover per individual per generation (as is typical for a single chromosome),
 and we are measuring tree lengths in units of generations,
-the mean number of ancestral recombination events that occur somewhere on the marginal trees
-per unit of genetic length.
+this is the mean number of ancestral recombination events that occur somewhere on the marginal trees.
 Any such ancestral recombination might change the marginal gene tree;
 if it does, it will add four new edges.
 Not all such recombinations actually change the tree topology,
-so at most we expect $16N\log(2N)$ new edges after those required to describe the first tree.
+so at most we expect $8N\log(N)$ new edges after those required to describe the first tree.
 Similarly, not every new edge derives from a never-before-seen node,
 but the number of nodes is at most equal to the number of edges plus the sample size.
-This reduces the initial storage of $120 N^2$ items to
```

Comment:
I didn't think we gained much by the level of precision here (plus, I'm not sure it was correct). To me this is the real high-level message that we want to convey.